### PR TITLE
Rename ENABLE_NODE_AUTOSCALER to ENABLE_CLUSTER_AUTOSCALER - part 1

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @piosz @fgrzadkowski @jszczepkowski

### CODE REVIEW

**Comment by piosz** [comment]
I think we should add some release-notes here.

**Comment by fgrzadkowski** [comment]
@mwielgus I agree with @piosz - we need release notes as it is a non-backward compatible change. It was an alpha feature, so that ok but we should document it.

File: cluster/aws/config-default.sh [hunk 1]
```diff
@@ -124,8 +124,8 @@ DNS_REPLICAS=1
 ENABLE_CLUSTER_UI="${KUBE_ENABLE_CLUSTER_UI:-true}"
 
 # Optional: Create autoscaler for cluster's nodes.
-ENABLE_NODE_AUTOSCALER="${KUBE_ENABLE_NODE_AUTOSCALER:-false}"
-if [[ "${ENABLE_NODE_AUTOSCALER}" == "true" ]]; then
+ENABLE_CLUSTER_AUTOSCALER="${KUBE_ENABLE_CLUSTER_AUTOSCALER:-false}"
+if [[ "${ENABLE_CLUSTEr_AUTOSCALER}" == "true" ]]; then
```
Comment (fgrzadkowski) [inline cluster/aws/config-default.sh]: s/r/R

File: cluster/common.sh [hunk 2]
```diff
@@ -673,9 +673,9 @@ RKT_PATH: $(yaml-quote ${RKT_PATH:-})
 KUBERNETES_CONFIGURE_CBR0: $(yaml-quote ${KUBERNETES_CONFIGURE_CBR0:-true})
 EOF
   fi
-  if [[ "${ENABLE_NODE_AUTOSCALER}" == "true" ]]; then
+  if [[ "${ENABLE_NODE_AUTOSCALER:-}" == "true" || "${ENABLE_CLUSTER_AUTOSCALER}" == "true" ]]; then
```
Comment (fgrzadkowski) [inline cluster/common.sh]: Should we add default value for `ENABLE_CLUSTER_AUTOSCALER`?

**Comment by fgrzadkowski** [comment]
Only some minor comments. In general LGTM. Please fix and let me know so I can add label.

File: cluster/common.sh (hunk 2 above)
Comment (mwielgus) [inline cluster/common.sh]: Not if we want to support ENABLE_NODE_AUTOSCALER for a bried time.

File: cluster/aws/config-default.sh (hunk 1 above)
Comment (mwielgus) [inline cluster/aws/config-default.sh]: Done.

**Comment by mwielgus** [comment]
This is not a breaking change old env variable is still supported. The next pr will be a breaking change and will have a release note.

**Comment by mwielgus** [comment]
This needs to be done before GKE changes. A follow up PR will be submitted once GKE changes are done.

**Pull request merged**